src/item.h

```diff
@@ -730,18 +736,23 @@ class item : public visitable<item>
         time_point last_rot_check = calendar::time_of_cataclysm;
 
         /**
-         * Calculate temperature differential and handle FROZEN/COLD/HOT states
+         * Calculate the thermal energy and temperature change of the item
          * @param temp Temperature of surroundings
          * @param insulation Amount of insulation item has
          * @param time Duration of time at which to process at temperature
          */
         void calc_temp( const int temp, const float insulation, const time_duration &time );
+        
+        /**
+         * Get the thermal energy of the item in Joules.
+         */
+        float get_item_thermal_energy();
 
-        /** Using item_tags and counters, calculate a static counter representation of the item's temperature */
-        int get_static_temp_counter() const;
-
-        /** Set temperature tags and counter according to a static counter */
-        void set_temp_from_static( const int counter );
+        /** Calculates item temperature (K) from energy (J)*/
+        float get_temp_from_energy( const float thermal_energy );
+        
```

Comment:
```suggestion

```
Or delete this line.

Reply:
Ohh I only ran the astyle on .cpp files.